### PR TITLE
Initial Landing Page

### PR DESCRIPTION
**Pull request by kentgottschalk**
Because

@bnferguson

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        margin-left: -50px; /* margin is -0.5 * dimension */
+        margin-top: -25px;
+    }​
+  </style>
+</head>
+
+<body>
+
+<div class="center">
+  <h1>GHOST</h1>
```
Comment (bnferguson) [inline index.html]: :ghost:

**Comment by bnferguson** [comment]
This looks great! :+1: :shipit:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<head>
+  <title></title>
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
+  <style>
+    html {background: #555555;}
+    h1 {color: #FFF; font-family: "Open Sans"; font-size: 30px;}
+    .center {
```
Comment (kentgottschalk) [inline index.html]: would be better to use flexbox

Comment (kentgottschalk) [inline index.html]: Red backgrounds is awesome

**Pull request merged**